config/manager/controller_manager_config.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+kind: ControllerManagerConfig
+health:
+  healthProbeBindAddress: :8081
+metrics:
+  bindAddress: 127.0.0.1:8080
+webhook:
+  port: 9443
+leaderElection:
+  leaderElect: true
+  resourceName: 06473caf.redhat.com
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)